config/hscConfig.py

```diff
@@ -1,9 +1,7 @@
 """
-Configure to force using astrometry_net instead of directly
-loading Pan-STARRS1 reference catalogs
-"""
+THE ASTROMETRY_NET_DATA NO LONGER EXISTS IN PIPE_TASKS see
 
-from lsst.pipe.tasks.setConfigFromEups import setPhotocalConfigFromEups, setAstrometryConfigFromEups
+"""
```

Comment:
See... what?

Reply:
ah yes I meant to add the ticket number .. which i did now

Reply:
I don't think we need a comment here to tell us the absence of something. The commit message would be sufficient and the merge commit would tell us the ticket number.